packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx

```diff
@@ -22,6 +22,7 @@ Defaults to true. If set to false, this rule will also report when a `switch` st
 
 When a `switch` statement over a union type is exhaustive, a final `default` case would be a form of dead code.
 Additionally, if a new value is added to the union type, a `default` would prevent the `switch-exhaustiveness-check` rule from reporting on the new case not being handled in the `switch` statement.
+If you want `default` to prevent `switch-exhaustiveness-check` when new values ​​are added to the union type, try using the `requireDefaultCaseForUnions` option
```

Comment:
how about
```suggestion
Additionally, if a new value is added to the union type and you're using `[considerDefaultExhaustiveForUnions](#considerDefaultExhaustiveForUnions)`, a `default` would prevent the `switch-exhaustiveness-check` rule from reporting on the new case not being handled in the `switch` statement.
```

or just omitting this check now that there is a recourse 🤷

Reply:
It seemed better to change it like the first suggestion, so we reflected it.